test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/Output/StringLiterals.cs

```diff
@@ -0,0 +1,238 @@
+#pragma checksum "StringLiterals.cshtml" "{ff1816ec-aa5e-4d10-87f7-6f4963833460}" "4bc2accb98c9c1f52bff719978da3f8449b7c9c6"
```

Comment:
This is one test of a brand new code path. Suggest adding a `CSharpCodeVisitorTest` or extending the `CSharpCodeGeneratorTest` class with a few unit tests at least for `Visit(ParentLiteralChunk)` and `Visit(LiteralChunk)`. Check near boundary conditions such as `string`s of length `0`, `1`, `255`, `256`, `257`, `1023`, `1024`, `1025`, ... Of course the middle few cases might be more interesting in the `CSharpCodeWriterTest` class.